controlplane/eks/api/v1beta1/awsmanagedcontrolplane_types.go

```diff
@@ -166,6 +167,10 @@ type AWSManagedControlPlaneSpec struct { //nolint: maligned
 	// Amazon VPC CNI addon or if you have specified a secondary CIDR block.
 	// +kubebuilder:default=false
 	DisableVPCCNI bool `json:"disableVPCCNI,omitempty"`
+
+	// AWSNodeEnvVars defines a list of environment variables to apply to the `aws-node` DaemonSet
+	// +optional
+	AWSNodeEnvVars []corev1.EnvVar `json:"awsNodeEnvVars,omitempty"`
```

Comment:
I'm wondering if we create a new struct called `VPCCNI` or `AWSNode` that has a field called `EnvVars` instead? 

We could then do a follow-up change  to add a `Disable` field and start deprecation of  `DisableVPCCNI` so that anything related to aws node/the vpc cni is on its own struct.

wdyt?

Reply:
Yes something like :+1:  We can then follow up later with:

```yaml
spec:
  VPCCNI:
    disable: true
```